docs/source/config/settings.rst

```diff
@@ -562,6 +562,14 @@ Emails
 
     Default: ``'indico.vendor.django_mail.backends.smtp.EmailBackend'``
 
+.. data:: EMAIL_LOG_STORE_ATTACHMENTS
```

Comment:
I would make this a `EMAIL_LOG_STORAGE` setting (default to `None`) that points to a backend to use for this purpose. `ATTACHMENT_STORAGE` is meant for valuable/long-term stuff like materials (and a bunch of other things but that's a different topic), so I'd rather not add log-related data there